src/configure

```diff
@@ -118,9 +119,14 @@
     (if prefix
       (string-append prefix "/bin/" prog)
       prog))
-  (list (make-patch (string-append "GERBIL_GSI=\"" (bin-path gambit-prefix "gsi") "\"\n")
-                    "GERBIL_GSI="
-                    '())))
+  `(,(make-patch (string-append "GERBIL_GSI=\"" (bin-path gambit-prefix "gsi") "\"\n")
+                 "GERBIL_GSI="
+                 '())
+    ,@(if install-prefix
+        (list (make-patch (string-append "GERBIL_HOME=\"" install-prefix "\"\n")
```

Comment:
does this mean we ignore the GERBIL_HOME variable as set by the user? I am not sure if that's a good idea.

Reply:
we should have a `DEFAULT_GERBIL_HOME` variable, which we patch and use it if `GERBIL_HOME` is unset.

Reply:
If you look at the scripts being edited, `gxi` and `gxi-build-script`, that's exactly what happens: we're changing the behavior in the case GERBIL_HOME is not defined.

Reply:
oh, good enough.